lib/feedback.js

```diff
@@ -21,8 +21,13 @@ module.exports = function (input,options) {
     var texvcres = texvcjs.check(input,options);
     if (texvcres.status !== '+') {
         out.success = false;
-        out.status = texvcres.status;
-        out.detail = texvcres.details;
+        out.error = {
```

Comment:
That's what I was thinking at the first place, but mathoid transforms it from error to detail
https://github.com/physikerwelt/mathoid-server/blob/c70ffd039949939f8c6df9ab32bbeb6b978d3842/routes/mathoid.js#L72